src/lib/y2network/config_reader/sysconfig_routes_reader.rb

```diff
@@ -114,9 +118,73 @@ def build_route(hash)
         Y2Network::Route.new(
           to:        hash["destination"] != DEFAULT_DEST ? build_ip(hash["destination"], mask) : :default,
           interface: iface,
-          gateway:   build_ip(hash["gateway"], MISSING_VALUE)
+          gateway:   build_ip(hash["gateway"], MISSING_VALUE),
+          options:   hash["extrapara"] || ""
         )
       end
+
+      # SCR agent for routes files definition
+      def ifroute_term(path)
```

Comment:
Not for now, but in the future we might want to use CFA (or something like that).

Reply:
yes, but this is tightly coupled with SCR and old good yast agents. Not useful elsewhere.